content/scripts/importer.js

```diff
@@ -72,12 +72,12 @@ async function populateRedirectInfo(pool, constraintsSQL, queryArgs) {
     // destination of each redirect that we'll keep.
     const isInfiniteLoop = chainOfRedirects.has(toUri);
     if (isInfiniteLoop) {
-      // This next URI in the redirect chain is already in
-      // the chain, so we've discovered an infinite loop.
-      const arrayOfRedirects = [...chainOfRedirects];
-      arrayOfRedirects.push(toUri);
-      const chainAsString = arrayOfRedirects.join(" --> ");
-      // console.error(`FOUND INFINITE REDIRECT LOOP: ${chainAsString}`);
+      // // This next URI in the redirect chain is already in
+      // // the chain, so we've discovered an infinite loop.
+      // const arrayOfRedirects = [...chainOfRedirects];
+      // arrayOfRedirects.push(toUri);
+      // const chainAsString = arrayOfRedirects.join(" --> ");
+      // // console.error(`FOUND INFINITE REDIRECT LOOP: ${chainAsString}`);
```

Comment:
Sorry. I can't remember why I commented this out. Can you forgive me????

Reply:
We can perhaps delete this code under `if (isInfiniteLoop) {`. I included it originally for crude reporting of infinite loops while developing. If we later think we need to report them, we can do it in a better way. So perhaps this entire `if-else` statement should collapse to a simple `if` statement?
```javascript
if (!isInfiniteLoop) {
  const nextUri = redirects.get(toUri);
  if (nextUri) {
    return extractFromChain(nextUri, chainOfRedirects.add(toUri));
  }
}
...
```

Reply:
If you say so, I'll do it.